### PR TITLE
kernel module update

### PR DESCRIPTION
**Pull request by crashdummymch**
in order to detect for /bin/true debian systems require to use --showconfig and some grep/sed magic

### CODE REVIEW

**Comment by chris-rock** [comment]
@crashdummymch Great finding! Does your approach apply to all linux or just debian/ubuntu ones?

**Comment by crashdummymch** [comment]
Just Debian based which is why the if statement is needed

File: libraries/linux_module.rb [hunk 1]
```diff
@@ -60,7 +60,11 @@ def version
   def command
     # Lets just ensure the last line in the kernel module's configuration is 'install /bin/true'
     # this is enough to be sure the module will not be loaded on next reboot or run of modprobe
-    modinfo_cmd = "/sbin/modprobe -n -v #{@module} | tail -n 1 | awk '{$1=$1;print}'"
+    modinfo_cmd = if inspec.os.redhat? || inspec.os.name == 'fedora'
+                    "/sbin/modprobe -n -v #{@module} | tail -n 1 | awk '{$1=$1;print}'"
+                  else
```
Comment (chris-rock) [inline libraries/linux_module.rb]: With this change we expect all linux other then redhat to behave like debian. At this point, it feels safer to do the if check for debian and use the redhat one as default. What do you think?

**Comment by crashdummymch** [comment]
I'm good either way I was just following the same president as code in
other places in the resource that uses the same logic

**Comment by rarenerd** [comment]
@crashdummymch I've tried to reproduce on Ubuntu 18.04 (bionic) and Debian 9.4 (stretch). Both support the original test with `/sbin/modprobe -n -v`. Can you provide more information about what was failing for you and on which versions?

**Comment by crashdummymch** [comment]
on ubuntu 16.0.4 if the module is not loaded it produces no result.  for inspec cis compliance it is required to ensure that the module is configured to be not installed and configured as /bin/true.  The only way I've found to do this is to utilize the --showconfig option in ubuntu.  If there is a better way please advise.

**Comment by rarenerd** [comment]
I've tested the CIS instructions to resolve this on an Ubuntu 16.04 host and it works as expected, so I'll be closing this PR.

For cramfs for example, adding `install cramfs /bin/true` to the modprobe configuration results in the expected output when using `/sbin/modprobe -n -v` and the control to succeed.

**Pull request closed without merging**